src/plib/resources/locales/en-US.php

```diff
@@ -42,7 +43,8 @@
     'syncAllDone' => 'All zones registered in Plesk were synchronized with Amazon.',
     'removeAllDone' => 'All zones hosted on Amazon were removed.',
     'awsApiAuthorization' => 'AWS API Authorization',
-    'awsDescriptionWindow' => 'To manage Route53, you need a restricted IAM user account to authenticate with Amazon. If you provide root account credentials, Plesk will create a restricted IAM user account for you.',
+    'awsDescriptionWindow' => 'To manage Route53, you need a restricted IAM user account to authenticate with Amazon.',
+    'awsDescriptionWindowRoot' =>' If you provide root account credentials, Plesk will create a restricted IAM user account for you.',
```

Comment:
Do "root account credentials" refer to Plesk or to Amazon?

Reply:
Root account refer to Amazon